src/gwc/src/test/java/org/geoserver/gwc/wms/CachingExtendedCapabilitiesProviderTest.java

```diff
@@ -122,6 +122,24 @@ public void testTileSets() throws Exception {
         assertXpathExists(tileSetPath + "[1]/Format", dom);
         assertXpathExists(tileSetPath + "[1]/Layers", dom);
         assertXpathExists(tileSetPath + "[1]/Styles", dom);
+
+        // Test RequireTiledParameter==false
```

Comment:
Confused, if the new setting affecting capabilities results at all?

Reply:
Direct Integration makes changes by adding in TileSets if tiled=true is sent, so I matched that. I suspect that no one actually uses that feature as it would need hand coding on the client side, but it seemed safest to match the existing behaviour

Reply:
Oh, I see... hum, that grows the capablities document quite a bit though. I also believe that nobody is really using that feature, the idea here is to use the cache just because clients happen to align to the grid. @simboss what do you think?